packages/@aws-cdk/toolkit-lib/lib/api/aws-cdk.ts

```diff
@@ -12,6 +12,7 @@ export { CloudWatchLogEventMonitor, findCloudWatchLogGroups } from '../../../../
 export { type WorkGraph, WorkGraphBuilder, AssetBuildNode, AssetPublishNode, StackNode, Concurrency } from '../../../../aws-cdk/lib/api/work-graph';
 export { Bootstrapper } from '../../../../aws-cdk/lib/api/bootstrap';
 export { loadTree, some } from '../../../../aws-cdk/lib/api/tree';
+export { createDiffChangeSet } from '../../../../aws-cdk/lib/api/deployments/cfn-api';
```

Comment:
i also need to use this function and there's not a good way of moving it to tmp-toolkit-helpers because i would basically be moving the entire `deployments` folder. wonder if this is ok for now

Reply:
Yes fine. Will move soon.

Reply:
ok, but doing so means that i export the `cfn-api` file out of `aws-cdk`